WalletWasabi.Gui/Global.cs

```diff
@@ -328,7 +328,15 @@ public async Task InitializeNoWalletAsync()
 
 				#endregion JsonRpcServerInitialization
 
-				WalletManager.RegisterServices(BitcoinStore, Synchronizer, Nodes, Config.ServiceConfiguration, FeeProviders, BitcoinCoreNode);
+				#region Blocks provider
+
+				var blockProvider = new CachedBlockProvider(
+					new P2pBlockProvider(Nodes, BitcoinCoreNode, Synchronizer, Config.ServiceConfiguration, Network), 
+					new FileSystemBlockRepository(blocksFolderPath, Network));
```

Comment:
Can we move the declaration of `blocksFolderPath` in line 125 to line 332?

Reply:
When you touch the Folder structure it is better to see them in one place. I would not move it.